tidb-cloud/tidb-cloud-import-local-files.md

```diff
@@ -74,3 +74,21 @@ Currently, this method supports importing one CSV file for one task into either
 9. After the import task is completed, you can click **Explore your data by Chat2Query** to query your imported data. For more information about how to use Chat2Qury, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md).
 
 10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.
+
+## FAQ
+
+### Can I only import some specified columns on TiDB Cloud?
+
+No. Currently, you can only import all columns of a CSV file into an existing table.
+
+In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:
+
+```sql
+CREATE TABLE `import_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  `address` varchar(64) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
+```
```

Comment:
```suggestion
```

If you use MySQL client and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.",  you can add `--local-infile=true` when connecting.
```

Reply:
why delete ```?

Reply:
With ```, the format is messy. Will add it later. Ignore it.